_data/flux-commands.yaml

```diff
@@ -119,7 +127,42 @@ commands:
             command: flux alloc -N 2 --queue=foo
           - title: Create an allocation in the background with two nodes on queue 'foo'
             command: flux alloc --bg -N 2 --queue=foo
-            
+
+  - name: flux bulksubmit
+    groups:
+      - name: Format String Examples
+        items:
+          - title: Pipe inputs into flux bulksubmit
+            command: seq 1 4 | flux bulksubmit echo {}
+          - title: Map the same inputs into flux bulksubmit
+            command: "flux bulksubmit echo {} ::: 1 2 3 4"
+          - title: Remove the extension of the input string (e.g., one)
+            command: "flux bulksubmit echo {.%} ::: one.txt two.txt three.txt"
+          - title: Get the basename of the input string (e.g., path.txt)
+            command: "flux bulksubmit echo {./} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Combine the two - get the basename with extension removed (e.g., path)
+            command: "flux bulksubmit echo {./%} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Get the directory name of the input string (e.g., /long/file)
+            command: "flux bulksubmit echo {.//} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Get the input sequence number, start counting at 0
+            command: "flux bulksubmit echo {seq} ::: 1 2 3 4"
+          - title: Get the input sequence number, start counting at 1 (you monster)
+            command: "flux bulksubmit echo {seq1} ::: 1 2 3 4"
+          - title: Define a custom counter called p2
+            command: "flux bulksubmit --dry-run --define=p2='2**int(x)' -n {.p2} hostname \
+   ::: $(seq 0 4)"
+
+      - name: Simple Examples
+        items:
+          - title: Compress every file under ./mydir in parallel, one process per core
+            command: find ./mydir | flux bulksubmit --wait xz {} 
+          - title: Do a dry run and provide multiple inputs
+            command: "flux bulksubmit --dry-run /bin/bash -c 'touch mydir/{0}; echo touched number {1}' ::: filea fileb filec filed ::: 1 2 3 4"
+          - title: Use carbon copy to submit identical jobs with different inputs
+            command: "flux bulksubmit --dry-run --cc={0} echo {1} ::: a b c ::: 0-1 0-3 0-7"
+          - title: Similar functionality with flux bulksubmit
+            command: "flux bulksubmit echo 'Hello I am job {}' ::: $(seq 1 10)"
```

Comment:
This one seems to be out of place. It says "similar functionality with `flux bulksubmit`", but the previous example also uses `bulksubmit`

Reply:
Good catch - this ordering has changed (so the context is outdated) - will try to write a better description!